community/vdr/APKBUILD

```diff
@@ -0,0 +1,186 @@
+# Contributor: Stefan Reiff <kroko87@hotmail.com>
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=vdr
+pkgver=2.4.0
+pkgrel=1
+pkgdesc="Video Disk Recorder"
+url="http://www.tvdr.de/"
+arch="all"
+license="GPL-2.0"
+depends=""
+depends_dev="libexecinfo-dev fontconfig-dev freetype-dev gettext-dev libjpeg-turbo-dev fribidi-dev
+	libcap-dev alsa-lib-dev libvdpau-dev libva-dev libx11-dev xcb-util-dev xcb-util-wm-dev
+	ffmpeg-dev libcdio-dev mpv"
```

Comment:
shouldn't it be mpv-dev ?

Reply:
mpv-dev isn't available, but package mpv contains the headers